dotCMS/src/main/java/com/dotcms/contenttype/model/field/FieldTypeFactory.java

```diff
@@ -0,0 +1,74 @@
+package com.dotcms.contenttype.model.field;
+
+import com.dotmarketing.exception.DotRuntimeException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.exception.DotRuntimeException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)